src/integration/appland/appMap/create.ts

```diff
@@ -13,10 +14,14 @@ export type CreateOptions = {
 
 export async function create(
   data: Buffer,
+  metadata?: Metadata,
```

Comment:
Maybe place this in CreateOptions since it’s optional?

Reply:
Thanks for the tip, I moved it into that interface.